src/Sylius/Behat/Client/ApiClientInterface.php

```diff
@@ -72,4 +73,13 @@ public function removeSubResource(string $subResource, string $id): void;
     public function updateRequestData(array $data): void;
 
     public function getLastResponse(): Response;
+
+    public function customRequest(
+        string $method,
+        string $uri,
+        array $parameters = [],
+        array $files = [],
+        array $headers = [],
+        string $content = null
+    ): Crawler;
```

Comment:
it shouldn't be Response?

Reply:
No, its delegated method from `Symfony\Component\BrowserKit\AbstractBrowser`->request, and it return Crawler, but if you need response, you can use getResponse()